packages/api/src/command/hie/unlink-patient-from-organization.ts

```diff
@@ -322,10 +326,37 @@ async function findAndInvalidateLinks(
       return;
     }
 
+    const patient = await getPatientOrFail({ cxId, id: patientId });
+    const cwAccess = await getCWAccessForPatient(patient);
+    if (cwAccess.error != null) throw new MetriportError(cwAccess.error);
+    const { commonWell, queryMeta } = cwAccess;
+
+    const downgradeRequests: Promise<NetworkLink>[] = [];
+    for (const link of invalidLinks.commonwell) {
+      if (!link._links?.downgrade?.href) continue;
+      const downgradeRequest = commonWell
+        .upgradeOrDowngradeNetworkLink(queryMeta, link._links.downgrade.href)
```

Comment:
Let's use `executeWIthNetworkRetries` so we retry on simple things like conn refused, timeout, etc.? [This PR](https://github.com/metriport/metriport/pull/3657)'s updating it across all CW integration.